### ISSUE TITLE
Another problem with multiple configurations, sorry.

### ISSUE THREAD

**Issue by Restiasan** [chiyadev/MudaeFarm#139]
Configuration servers load as intended but the second instance doesn't detect rolls, only the first one does.

**Comment by phosphene47** [comment]
That's not right. Configuration servers are completely isolated so it must be a misconfiguration on the second server.

If not, can you enable verbose logging and attach the log file?

**Comment by Restiasan** [comment]
How do I enable verbose logging?
I tried creating a new profile and used the default config but got the same results.

**Comment by Restiasan** [comment]
When I try to run it with verbose it closes immediately.
I also tried creating a new profile and running it with default configuration but got the same result.

**Comment by Restiasan** [comment]
After trying for a while it seems that the bot doesn't work on a specific channel. 
Both instances work well on my main channel it doesn't work on the secondary channel.

**Comment by Restiasan** [comment]
After testing, it seems that the bot doesn't recognize channels that are private (for specific roles only).

**Comment by phosphene47** [comment]
Verbose logging should be working now.

**Comment by Restiasan** [comment]
Multiple Server Test Log + VID https://youtu.be/vu6udWUNdfs
[log_2020-06-04 20.19.53Z-20200604.txt](https://github.com/chiyadev/MudaeFarm/files/4733138/log_2020-06-04.20.19.53Z-20200604.txt)

Normal Log on a single channel (that doesn't work)
[log_2020-06-04 20.22.20Z-20200604.txt](https://github.com/chiyadev/MudaeFarm/files/4733141/log_2020-06-04.20.22.20Z-20200604.txt)

Settings (I tested it before with default settings so I don't think they are relevant)
![Captura de pantalla (1184)](https://user-images.githubusercontent.com/64933517/83820132-29895400-a6a2-11ea-9355-a47b5076b43b.png)

**Comment by Restiasan** [comment]
![IMG_20200625_070038](https://user-images.githubusercontent.com/64933517/85699166-ad3acd00-b6b1-11ea-9da8-257502c1f0a5.jpg)
From video, I'm actually using 3.1.1

**Comment by phosphene47** [comment]
Hmm, it seems like `ulong.Parse` is respecting your system culture which is Spanish, but I'm not sure about the numerical formatting differences between Spanish and English when channel ID isn't supposed to contain decimals.

Nevertheless, I'll try fixing all number parsing code to be culture invariant in the next release.

**Comment by Restiasan** [comment]
Oh, that reminds me that I'm using Japanese Language for non-unicode settings, I'll try with the default later.
